lib/pages/AthletePage.dart

```diff
@@ -378,7 +388,8 @@ class _AthletePageState extends State<AthletePage> {
                                                   style: textStyle(Colors.red,
                                                       12, false, false))),
                                         ]),
-                                        Text("${athlete.war.toStringAsFixed(2)} AX",
+                                        Text(
```

Comment:
[nit](https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak): I don't see why this needed to be changed.